distribution/std-lib/Standard/src/Base/Data/Any/Extensions.enso

```diff
@@ -45,61 +64,117 @@ Any.!= that = (this == that).not
 Any.> : Any -> Boolean
 Any.> that = this.compare_to that == Ordering.Greater
 
+## Checks if `this` is greater than or equal to `that`.
+
+   Arguments:
+   - that: The value to compare `this` against.
+
+   To have `>=` defined, a type must define both `>` and `==`.
+
+   ! Implementing Greater Than or Equal
+     In general we recommend that you do not implement this by hand to avoid
+     semantic incorrectness. It is very rare that a more efficient
+     implementation can be provided than the one given here.
+
+   > Example
+     Checking if 1 is greater than or equal to 10.
+         1 >= 10
+Any.>= : Any -> Boolean
+Any.>= that = (this > that) || (this == that)
+
 ## Checks if `this` is less than `that`.
 
+   Arguments:
+   - that: The value to compare `this` against.
+
    To have `<` defined, a type must define `compare_to`, returning an Ordering.
 
+   ! Implementing Less Than
+     Many types can admit a definition of less than that is more efficient than
+     the generic one given here. When implementing this for your own types
+     please ensure that it is semantically equivalent to this definition.
+
    > Example
-     Compare two integers.
-         1 < 10 == True
+     Checking if 1 is less than 10.
+         1 < 10
 Any.< : Any -> Boolean
 Any.< that = this.compare_to that == Ordering.Less
 
+## Checks if `this` is less than or equal to `that`.
+
+   Arguments:
+   - that: The value to compare `this` against.
+
+   To have `<=` defined, a type must define both `<` and `==`.
+
+   ! Implementing Less Than or Equal
+     In general we recommend that you do not implement this by hand to avoid
+     semantic incorrectness. It is very rare that a more efficient
+     implementation can be provided than the one given here.
+
+   > Example
+     Checking if 1 is less than or equal to 10.
+         1 <= 10
+Any.<= : Any -> Boolean
+Any.<= that = (this < that) || (this == that)
+
 ## Checks if the type is an instance of `Nothing`.
 
+   Nothing in Enso is used as a universal value to indicate the lack of presence
+   of a value. This function is primarily useful in the IDE.
+
    > Example
-     Checking if a variable `a` is nothing.
-         a.is_nothing
+     Checking if the value 1 is nothing.
+         1.is_nothing
 Any.is_nothing : Boolean
 Any.is_nothing = case this of
     Nothing -> True
     _ -> False
 
-## Executes the provided handler on a dataflow error, or executes as identity on
-   a non-error value.
+## Executes the provided handler on a dataflow error, or returns a non-error
+   value unchanged.
 
    Arguments:
    - handler: The function to call on this if it is an error value. By default
      this is identity.
 
    > Example
-     Catching an erroneous value to perform some operation on it.
-       (Time.Time_Error "Message").catch (err -> IO.println err)
+     Catching an erroneous value and getting the length of its message.
+       (Time.Time_Error "Message").catch (err -> IO.println err.error_message)
```

Comment:
I don't see where we do any `length` related operations here. We print the message.